PWGHF/TableProducer/HFDplusToPiKPiCandidateSelector.cxx

```diff
@@ -119,13 +119,18 @@ struct HFDplusToPiKPiCandidateSelector {
     // looping over 3-prong candidates
     for (auto& candidate : candidates) {
 
-      // final selection flag: 0 - rejected, 1 - accepted
+      // final selection flag:
+      // 0 - rejected
+      // BIT(0) - bit for D+ candidate
+      // BIT(1) - selected by topological cuts
+      // BIT(2) - selected by PID cuts
```

Comment:
Since these will be shared among taks, I would be in favour of defining them in a dedicated `enum` in `HFCandidateSelectionTables.h`.
Something like `SelectionSteps`.